valhalla/common/test_helpers.py

```diff
@@ -34,6 +34,8 @@ def setUp(self):
         self.time_patcher = patch('valhalla.userrequests.serializers.timezone.now')
         self.mock_now = self.time_patcher.start()
         self.mock_now.return_value = datetime(2016, 9, 1, tzinfo=timezone.utc)
+        super().setUp()
```

Comment:
Agreed. But we'll have to go through all our tests and call the setUp methods explicitly.